lua/DCS-gRPC/methods/mission.lua

```diff
@@ -411,3 +413,59 @@ GRPC.onDcsEvent = function(event)
     return nil
   end
 end
+
+function missionCommandCallback(callbackId)
+  GRPC.event({
+    time = timer.getTime(),
+    event = {
+      type = "missionCommand",
+      id = callbackId,
+    }
+  })
+  return GRPC.success(nil)
+end
+
+GRPC.methods.addMissionCommand = function(params)
```

Comment:
> I'd argue missionCommand's is actually one of the more clean ED API's, but providing a one-to-one mapping ends up being very messy and verbose. I'm curious what people think, but to me this abstraction seems cleaner.

I do not believe it is that messy or verbose and a bit of verbosity in the server-side code is preferable to having to make clients written in different languages dea with extra protocol complexity with `oneofs` and the like.

I prefer we keep everything as simple as possible and split the APIs to, in this case, follow the ED style AddMissionCommandForCoalition etc.